POSserver/POSserver/schema.py

```diff
@@ -0,0 +1,34 @@
+import graphene
+import graphql_jwt
+import server.schemas
+
+
+class SuperQuery(
+    server.schemas.tag.Query,
+    server.schemas.note.Query,
+    server.schemas.contractor.Query,
+    server.schemas.user.Query,
+    server.schemas.part.Query,
+    server.schemas.client.Query,
+    server.schemas.job.Query,
+    graphene.ObjectType,
+):
+    pass
+
+
```

Comment:
I like the SuperQuery code...I'd get rid of the orphaned 'pass' on line 16

Reply:
This pass is not orphaned.  SuperQuery has no methods, so 'pass' is necessary syntactically.